src/kOS/Execution/Stack.cs

```diff
@@ -8,7 +8,7 @@ namespace kOS.Execution
 {
     public class Stack : IStack
     {
-        private const int MAX_STACK_SIZE = 1000;
+        private const int MAX_STACK_SIZE = 3000;
```

Comment:
I noticed it wasn't counting the "above stack" in this before, so when I fixed that it also effectively reduced stack capacity, so I increased it to compensate.